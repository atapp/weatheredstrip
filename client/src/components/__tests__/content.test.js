import React from 'react';
import { render, cleanup } from 'react-testing-library'
import ReactDOM from 'react-dom';
import Content from '../content.react';
import 'jest-dom/extend-expect';
import testData from './test_data.json';
afterEach(cleanup)

// Full content is not tested yet, because timestamp is dependent on current time.
it('Content without renders properly', () => {
  const { container } = render(<Content />)
  const content = container.firstChild
  expect(content).toMatchSnapshot()
});
