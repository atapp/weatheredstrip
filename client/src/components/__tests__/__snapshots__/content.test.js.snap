// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content without renders properly 1`] = `
<div
  class="Content"
>
  <div
    class="user-content fullborder"
  >
    <p>
      Please use the searchbox to initate a search.
    </p>
  </div>
</div>
`;
